src/components/calcite-time-picker/calcite-time-picker.tsx

```diff
@@ -0,0 +1,702 @@
+import {
+  Component,
+  Element,
+  Host,
+  h,
+  Prop,
+  VNode,
+  Event,
+  EventEmitter,
+  Watch,
+  State,
+  Listen,
+  Method
+} from "@stencil/core";
+import { Scale, Theme } from "../interfaces";
+import { AmPm, formatNumberAsTimeString, MinuteOrSecond, numberKeys, Time } from "./utils";
+
+@Component({
+  tag: "calcite-time-picker",
+  styleUrl: "calcite-time-picker.scss",
+  shadow: true
+})
+export class CalciteTimePicker {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+
+  @Element() el: HTMLCalciteTimePickerElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  //--------------------------------------------------------------------------
+
+  /** The hour value (24-hour format) */
+  @Prop({ reflect: true, mutable: true }) hour?: string = "--";
+
+  /** Format of the hour value (12-hour or 24-hour) (this will be replaced by locale eventually) */
+  @Prop({ reflect: true }) hourDisplayFormat: "12" | "24" = "12";
```

Comment:
Since this only has 2 options, maybe a bool would be better here? like `military`?

Reply:
This prop is going to get replaced by `locale` anyway.  This is just a stop-gap to support the rendering of both modes in the meantime.